### PR TITLE
[Fix] defaults detrend locreg

### PR DESCRIPTION
**Pull request by danibene**
# Description

See https://github.com/neuropsychology/NeuroKit/issues/728

# Proposed Changes

I added `sampling_rate` as an argument to the `signal_detrend()` function.

# Checklist

Here are some things to check before creating the PR. If you encounter any issues, do let us know :)

- [x] I have read the [CONTRIBUTING](https://github.com/neuropsychology/NeuroKit/blob/master/.github/CONTRIBUTING.rst#structure-and-code) file.
- [x] My PR is targetted at the **dev branch** (and not towards the master branch).

### CODE REVIEW

File: neurokit2/signal/signal_detrend.py [hunk 1]
```diff
@@ -44,15 +45,19 @@ def signal_detrend(
...
-        desired low frequency band to remove multiplied by the sampling rate
-        (for instance, ``1.5*1000`` will remove frequencies below 1.5Hz for a signal sampled at
-        1000Hz).
+        desired low frequency band to remove multiplied by the sampling rate if sampling_rate is not
+        provided to the function directly (for instance, ``1.5*1000`` will remove frequencies below 
+        1.5Hz for a signal sampled at 1000Hz).
     stepsize : float
         Only used if ``method`` is ``"locreg"``. Similarly to ``window``, ``stepsize`` should also
-        be multiplied by the sampling rate.
+        be multiplied by the sampling rate if sampling_rate is not provided.
```
Comment (DominiqueMakowski) [inline neurokit2/signal/signal_detrend.py:53]: if ~not~ provided?

File: neurokit2/signal/signal_detrend.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ def signal_detrend(
     window=1.5,
     stepsize=0.02,
     components=[-1],
+    sampling_rate=None,
```
Comment (DominiqueMakowski) [inline neurokit2/signal/signal_detrend.py:18]: we should put it to 1000 by default given that default stepsize is not an integer, 

also we should probably pipe down the sampling rate in the routines where we use it (ecg_process?)

Comment (danibene) [reply neurokit2/signal/signal_detrend.py]: currently there don't seem to be any functions that call the locreg method of detrending (or have a parameter for specifying the method of detrending), other than intervals_process which I changed

File: neurokit2/signal/signal_detrend.py (hunk 1 above)
Comment (danibene) [reply neurokit2/signal/signal_detrend.py]: I just left it out completely now, feel free to change if you think it should be specified

**Review by DominiqueMakowski — APPROVED** [review]

**Pull request merged**